.travis.yml

```diff
@@ -54,7 +54,18 @@ matrix:
     - compiler: clang
     - os: osx
   include:
-    - env: BUILD_TYPE="Debug" WITH_BFD="yes" WITH_LATEST_GCC="yes" TEST_CLANG_FORMAT="yes" WITH_COVERAGE="yes" WITH_MPC="yes"
+    - env: BUILD_TYPE="Debug" WITH_BFD="yes" WITH_LATEST_GCC="yes" WITH_COVERAGE="yes" WITH_MPC="yes"
+      compiler: gcc
+      os: linux
+      addons:
+        apt:
+          sources:
+          - ubuntu-toolchain-r-test
+          - llvm-toolchain-precise-3.7
```

Comment:
removing any one from these sources is resulting in error , See [1](https://travis-ci.org/ranjithkumar007/symengine/jobs/216311941/config) , [2](https://travis-ci.org/ranjithkumar007/symengine/jobs/216292580/config).